airbyte-webapp/src/components/ui/ResizablePanels/ResizablePanels.module.scss

```
@@ -0,0 +1,59 @@
+@use "scss/colors";
+@use "scss/variables";
+
+/*
+* This eliminates flickering scrollbars when resizing using the reflex splitter.
+* Class names need to be repeated so that it overrides default reflex styles.
+* See more details in https://github.com/airbytehq/airbyte/pull/15996#issuecomment-1229019827
+*/
+.panelStyle.panelStyle {
+  overflow: hidden;
+}
+
+.panelContainer {
+  overflow: auto;
+  height: 100%;
+}
+
+.lightOverlay {
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  z-index: 10;
+  background-color: colors.$white;
+  overflow: hidden;
+  padding-top: 50px;
```

```suggestion
  padding-top: variables.$spacing-2xl;
```

`variables.$spacing-2xl` is 40px, but maybe better to conform to the spacing system here